<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>appPreferencesDialog</class>
 <widget class="QDialog" name="appPreferencesDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Умолчания</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_14">
   <property name="margin">
    <number>4</number>
   </property>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabMain">
      <attribute name="title">
       <string>Основные настройки</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="8" column="2">
        <widget class="CRBComboBox" name="cmbResult">
         <property name="whatsThis">
          <string>результат осмотра</string>
         </property>
         <item>
          <property name="text">
           <string>Результат</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="CRBComboBox" name="cmbEventTypeAdult"/>
       </item>
       <item row="0" column="2">
        <widget class="CPolyclinicComboBox" name="cmbOrganisation"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblOrganisation">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;ЛПУ</string>
         </property>
         <property name="buddy">
          <cstring>cmbOrganisation</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="CKLADRComboBox" name="cmbDefaultCity"/>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="btnSelectOrganisation">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="lblDiagnosticResult">
         <property name="text">
          <string>Результат осмотра</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>cmbDiagnosticResult</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="CRBComboBox" name="cmbDiagnosticResult">
         <property name="whatsThis">
          <string>результат осмотра</string>
         </property>
         <item>
          <property name="text">
           <string>Результат</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblOrgStructure">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;Подразделение</string>
         </property>
         <property name="buddy">
          <cstring>cmbOrgStructure</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblDefaultProvince">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;Область</string>
         </property>
         <property name="buddy">
          <cstring>cmbProvince</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblDefaultCity">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;Нас.пункт по умолчанию</string>
         </property>
         <property name="buddy">
          <cstring>cmbDefaultCity</cstring>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>61</width>
           <height>174</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="lblResult">
         <property name="text">
          <string>Результа&amp;т</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>cmbResult</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lblEventTypeAdult">
         <property name="text">
          <string>Тип события взрослый</string>
         </property>
         <property name="buddy">
          <cstring>cmbEventTypeAdult</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblPerson">
         <property name="text">
          <string>&amp;Врач</string>
         </property>
         <property name="buddy">
          <cstring>cmbPerson</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="CKLADRComboBox" name="cmbProvince"/>
       </item>
       <item row="5" column="2">
        <widget class="CDbComboBox" name="cmbContract"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblContract">
         <property name="text">
          <string>&amp;Договор</string>
         </property>
         <property name="buddy">
          <cstring>cmbOrgStructure</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="CPersonComboBoxEx" name="cmbPerson"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="lblEncoding">
         <property name="text">
          <string>Кодировка DBF файла</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>cmbEncoding</cstring>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QComboBox" name="cmbEncoding">
         <item>
          <property name="text">
           <string>cp1251</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cp866</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lblEventTypeChild">
         <property name="text">
          <string>Тип события детский</string>
         </property>
         <property name="buddy">
          <cstring>cmbEventTypeChild</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="CRBComboBox" name="cmbEventTypeChild"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPolyclinicComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CKLADRComboBox</class>
   <extends>QComboBox</extends>
   <header>KLADR/kladrComboxes.h</header>
  </customwidget>
  <customwidget>
   <class>CDbComboBox</class>
   <extends>QComboBox</extends>
   <header>library/DbComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbOrganisation</tabstop>
  <tabstop>btnSelectOrganisation</tabstop>
  <tabstop>cmbOrgStructure</tabstop>
  <tabstop>cmbDefaultCity</tabstop>
  <tabstop>cmbProvince</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>appPreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>674</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>appPreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>674</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
